terracumber_config/tf_files/Uyuni-Master-build-validation-PRV.tf

```diff
@@ -1518,6 +1520,24 @@ module "sles15sp4-terminal" {
     manufacturer       = "HP"
     product            = "ProLiant DL360 Gen9"
   }
+  private_ip         = 6
+  private_name       = "sle15sp4terminal"
+}
+
+module "dhcp-dns" {
+  source             = "./modules/dhcp_dns"
+  base_configuration = module.base_retail.configuration
+  name               = "dhcp-dns"
+  image              = "opensuse155o"
+  private_hosts = [
+    module.sles12sp5-terminal.configuration,
```

Comment:
also the proxy

Reply:
We do not use the containerized proxy in the Uyuni BV right now. There is no definition in this `.tf` file. So for now we will use the regular traditional proxy.

Reply:
If so, you don't need the DHCP and DNS server. With a traditional proxy it's the proxy that does DHCP and DNS, via formulas.

(I agree that requesting for the new variables of the terminals in this context is a bit stupid, as they won't be used.)

Remove the dhcp-dns section then?

Reply:
Ok, makes sense. Then I leave only the mandatory values there, like e.g.
```tf
  private_ip         = 5
  private_name       = "sle12sp5terminal"
```

Reply:
yes, even though they are useless - sorry

Reply:
No worries. Better this way than hiding them with default values.